src/battle-dex.ts

```diff
@@ -367,7 +367,7 @@ const Dex = new class implements ModdedDex {
 		return ability;
 	}
 
-	getTemplate(nameOrTemplate: string | Template | null | undefined): Template {
+	getTemplate(nameOrTemplate: string | Template | null | undefined, isDoubles: boolean = false): Template {
```

Comment:
This signature doesn't match the signature in the server, and it's a digression I wouldn't want to maintain.

It would make more sense to support `gen7doubles` etc as a mod name internally. Another option would be to have separate `tier` and `doublesTier` properties on the Template object.

Reply:
Both are probably good ideas: `doublesTier` as a teambuilder-table property, and `gen7doubles` as a modid.

Reply:
Alright yeah. I had mentioned doing it those ways in the dev room as well. Will do when I get the chance! :+1: